cmd/snap-repair/runner.go

```diff
@@ -984,6 +984,9 @@ func (run *Runner) Verify(repair *asserts.Repair, aux []asserts.Assertion) error
 		trustedBS.Put(asserts.AccountKeyType, t)
 	}
 	for _, t := range sysdb.Trusted() {
+		// we do *not* add the defalt sysdb trusted account
+		// keys here because the repair assertions have their
+		// own root of trust
```

Comment:
own \*dedicated\* root of trust

Reply:
Thanks for this suggestion - I updated the comment and force pushed.